cmd/cloud-controller-manager/controller-manager.go

```diff
@@ -48,9 +48,9 @@ func main() {
 	// normalize func and add the go flag set by hand.
 	// utilflag.InitFlags()
 	logs.InitLogs()
-	defer logs.FlushLogs()
```

Comment:
I understand that os.Exit is causing the defer to be skipped as this is main and os.Exit in main means main never returns. However it is not guaranteed that we will go down the path leading to the os.Exit and for those cases we are not not flushing the logs. The best solution would seem to be leave this defer here and add a comment explaining why we have both.

Reply:
Yes,  when `command.Execute()` not return error,  `FlushLogs()` will not run.

but keep two `logs.FlushLogs()` may not be the best solution.  I take the `os.Exit()` in defer can handle it.

Please review it again. Thanks.

Reply:
defer is LIFO so that should work.